src/gui/rss/automatedrssdownloader.cpp

```diff
@@ -308,10 +309,10 @@ RssDownloadRulePtr AutomatedRssDownloader::getCurrentRule() const
 void AutomatedRssDownloader::initLabelCombobox()
 {
   // Load custom labels
-  const QStringList customLabels = Preferences::instance()->getTorrentLabels();
-  foreach (const QString& label, customLabels) {
-    ui->comboLabel->addItem(label);
-  }
+  QStringList customLabels = Preferences::instance()->getTorrentLabels();
+  std::sort(customLabels.begin(), customLabels.end(), (bool (*)(const QString&,const QString&))misc::naturalCmp);
```

Comment:
@glassez, I was fixing the `naturalCmp()`, a `commit --amend` seems to wash out your notes (the type cast concern), sorry about that...